include/dbmanager.h

```diff
@@ -49,6 +49,12 @@ class DbManager
      * @param match : The required match.
      */
 	bool extractMatch(const int& matchId, Match& m);
+
+    bool fillMatch(const int& id,
+                   const int& hTName,
+                   const int& aTName,
+                   const int& hTScore,
+                   const int& aTScore);
```

Comment:
on all members you forgot the a before the name. Plus the scores should not be const because when a match is not played there is no score yet and you need to update them after the match is played ;)

Reply:
These are the method parameters and not the class attributes. So they are not submitted to this rule.
I fill these values into database, I do not have any intention to change their value.

Reply:
My bad ! I did not see the line on which the method name was declared